src/i18n/translations.js

```diff
@@ -23,6 +23,8 @@ export const resources = {
         askQuestion: 'Ask a Question',
         openBroadcast: 'Open Broadcast',
         closeBroadcast: 'Close Broadcast',
+        write: 'Write',
```

Comment:
Add placeholders (with English) for Spanish and Russian languages.
Add proper strings for Hebrew

Reply:
Do this for all messages.